### PR TITLE
fix: add support for negative stepsize, cleanup

### PR DESCRIPTION
**Pull request by afonsofonbraga**
Furuno rotates counterclockwise, with this PR now negative stepsizes are supported

### CODE REVIEW

File: tasks/EchoesToFrameConverterTask.cpp [hunk 1]
```diff
@@ -99,21 +100,27 @@ void EchoesToFrameConverterTask::addEchoesToFrame(Radar const& echo,
...
             echo.sweep_data.begin() + (current_angle + 1) * echo.sweep_length,
-            m_echoes.begin() + echo.sweep_length * ((start_angle_unit + current_angle) %
-                                                       angles_in_a_frame));
+            m_echoes.begin() + echo.sweep_length * echo_position);
     }
 }
 
 void EchoesToFrameConverterTask::publishFrame()
 {
+    m_cv_frame = 0;
```
Comment (DeboraTahara) [inline tasks/EchoesToFrameConverterTask.cpp:123]: Why is it global?

**Review by DeboraTahara — APPROVED** [review]

**Review by higor-melo — APPROVED** [review]

File: tasks/EchoesToFrameConverterTask.cpp (hunk 1 above)
Comment (afonsofonbraga) [reply tasks/EchoesToFrameConverterTask.cpp:123]: it is faster just to erase the cv matrix instead of creating a new one.

**Review by doudou — APPROVED** [review]

**Comment by doudou** [comment]
asking 2/3 of the team for review is a bit  ... too much ?

**Comment by afonsofonbraga** [comment]
> asking 2/3 of the team for review is a bit ... too much ?
I added the trainees to follow up not as reviewers... next time I'll add just the reviewers.

**Pull request merged**